modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/SelectionFilter.js

```diff
@@ -302,55 +346,48 @@ function SelectionFilter({
 						className="inline-scroller mx-n2 px-2"
 						ref={setScrollingArea}
 					>
-						{items.map((item) => {
-							const itemValue = itemKey
-								? item[itemKey]
-								: item.value;
-							const itemLabel = itemLabelProp
-								? getValueFromItem(item, itemLabelProp)
-								: item.label;
+						{items.map(({label, value}) => {
 							const newValue = {
-								label: itemLabel,
-								value: itemValue,
+								label,
+								value,
```

Comment:
Nice cleanup! After this, we don't need `itemLabelProp` alias, it can be just `itemLabel`.